internal/apps/dop/providers/issue/core/query/convert.go

```diff
@@ -51,6 +51,20 @@ func (p *provider) Convert(model dao.Issue, identityInfo *commonpb.IdentityInfo)
 	for _, v := range subscribers {
 		issue.Subscribers = append(issue.Subscribers, v.UserID)
 	}
+	if identityInfo != nil && identityInfo.OrgID != "" {
+		orgID, err := strconv.ParseUint(identityInfo.OrgID, 10, 64)
+		if err != nil {
+			return nil, err
+		}
+		propertyInstances, err := p.GetIssuePropertyInstance(&pb.GetIssuePropertyInstanceRequest{
```

Comment:
Get OrgID from identityInfo is incorrect for internal invoke. You should get from issue model.

And you should consider the performance, properties is not required in all situations.

Reply:
issue model doesn't contain `OrgID`, only browser request needs property user.  and only query issue detail from browser will get properties, batch query issues will not query properties. so it shouldn't increase a lot of performance consumption